### PR TITLE
Optimize zshrc

### PR DESCRIPTION
**Pull request by DanKGooGLy**

### CODE REVIEW

**Comment by DanKGooGLy** [comment]
Wait what

File: .config/zsh/.zshrc [hunk 1]
```diff
@@ -1,7 +1,3 @@
-for file in "${XDG_CONFIG_HOME:-~/.config}"/sh/rc.d/**/*; do
+for file in "${XDG_CONFIG_HOME:-$HOME/.config}/sh/rc.d/**/*" "$ZDOTDIR/rc.d/**/*.zsh"; do
 	source "$file"
-done
-
-for file in "$ZDOTDIR"/rc.d/**/*.zsh; do
-	source "$file"
-done
+done; unset file
```
Comment (33kk) [inline .config/zsh/.zshrc:3]: Is that unset necessary :thonk:

Comment (DanKGooGLy) [reply .config/zsh/.zshrc:3]: Yes because since your zshrc is sourcing it there's gonna be a `$file` variable, check it yourself with `echo $file`

**Comment by DanKGooGLy** [comment]
What the hell

**Comment by 33kk** [comment]
WAYTOODANK

**Pull request closed without merging**